ERCS/erc-7813.md

```diff
@@ -0,0 +1,498 @@
+
+#### Store_SpliceDynamicData
+
+Splicing the dynamic data of a Record involves modifying the packed encoded representation of its dynamic fields by removing, replacing, and/or inserting new bytes in place.
+
+The `Store_SpliceDynamicData` event MUST be emitted to signal that the dynamic data of the Record has been spliced.
+
+```solidity
+event Store_SpliceDynamicData(
+  ResourceId indexed tableId,
+  bytes32[] keyTuple,
+  uint8 dynamicFieldIndex,
+  uint48 start,
+  uint40 deleteCount,
+  EncodedLengths encodedLengths,
+  bytes data
+);
+```
+
+Parameters:
+
+| **Name**          | **Type**       | **Description**                                                                                                                                          |
+| ----------------- | -------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| tableId           | ResourceId     | The ID of the table where the data is spliced                                                                                                            |
+| keyTuple          | bytes32[]      | An array representing the composite key for the record                                                                                                   |
+| dynamicFieldIndex | uint8          | The index of the dynamic field to splice data, relative to the start of the dynamic fields (Dynamic field index = field index - number of static fields) |
+| start             | uint48         | The start position in bytes for the splice operation                                                                                                     |
+| deleteCount       | uint40         | The number of bytes to delete in the splice operation                                                                                                    |
+| encodedLengths    | EncodedLengths | The resulting encoded lengths of the dynamic data of the record                                                                                          |
+| data              | bytes          | The data to insert into the dynamic data of the record at the start byte                                                                                 |
+
+#### Store_DeleteRecord
+
+The `Store_DeleteRecord` event MUST be emitted to signal that the Record has been deleted from the Table.
+
+```solidity
+event Store_DeleteRecord(ResourceId indexed tableId, bytes32[] keyTuple);
+```
+
+Parameters:
+
+| **Name** | **Type**   | **Description**                                        |
+| -------- | ---------- | ------------------------------------------------------ |
+| tableId  | ResourceId | The ID of the table where the record is deleted        |
+| keyTuple | bytes32[]  | An array representing the composite key for the record |
+
+#### Example
```

Comment:
This should probably be moved to your Reference Implementation section. You can link to it from here though.